test/integration/targets/ec2_asg_scheduled_action/tasks/main.yml

```diff
@@ -0,0 +1,203 @@
+---
+# tasks file for test_ec2_asg_scheduled_action
+
+- name: Test incomplete credentials with ec2_asg_scheduled_action
+
+  module_defaults:
+    ec2_asg_scheduled_action:
+        scheduled_action_name: "{{ scheduled_action_name }}"
+        autoscaling_group_name: "{{ autoscaling_group_name }}"
+        desired_capacity: "{{ desired_capacity }}"
+        recurrence: "{{ recurrence }}"
+
+  block:
+
+    # ============================================================
+
+    - name: test invalid profile
```

Comment:
@jillr Given this is an AnsibleAWSModule module do you think there's any real value testing the standard AWS args?

Reply:
Perhaps not, but I will always take more/slightly redundant tests - better to have the CYA IMO.